extensions/components/com_redcore/admin/layouts/joomla/form/renderfield.php

```diff
@@ -38,7 +38,7 @@
 				<?php echo $label ?>
 				</div>
 			<?php endif; ?>
-			<div class="col-md-9 col-sm-8">
+			<div class="col-md-12 col-sm-8">
```

Comment:
what if you have title? Then you will have all fields in 2 separate lines...
https://github.com/LouiseDissing/redCORE/blob/77493c94751cabed59d1f4c8a0a90520e231fc76/extensions/components/com_redcore/admin/layouts/joomla/form/renderfield.php#L36